### PR TITLE
[ko] translate the glossary term eviction reviewed

### PR DESCRIPTION
**Pull request by tonyhan18**
I take my feedback all.
Thanks!!

### CODE REVIEW

**Comment by seokho-son** [comment]
Hi @tonyhan18 

This PR has two individual commits. I think the commit about cAdvisor is a mistake. Please remove the commit from this PR. Also, #35901 is a similar case.

[[ko] translate the glossary term eviction](https://github.com/kubernetes/website/pull/35899/commits/538573343ce76199fc7a4b979261e364a8f811cb)


[[ko] translate the glossary term cAdvisor](https://github.com/kubernetes/website/pull/35899/commits/4e1d2bd0871905d656c76d42e247def6d80d00f0)

File: content/ko/docs/reference/glossary/cadvisor.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+date: 2021-12-09
+full_link: https://github.com/google/cadvisor/
+short_description: >
+  Tool that provides understanding of the resource usage and performance characteristics for containers
+  컨테이너에 대한 리소스 사용량 및 성능 특징을 이해하게끔 돕는 도구
+aka:
+tags:
+- tool
+---
+cAdvisor (Container Advisor) provides container users an understanding of the resource usage and performance characteristics of their running {{< glossary_tooltip text="containers" term_id="container" >}}.
```
Comment (bart2001) [inline content/ko/docs/reference/glossary/cadvisor.md:13]: 영어 원문 삭제가 필요합니다.

File: content/ko/docs/reference/glossary/cadvisor.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+cAdvisor(Container Advisor)는 컨테이너 사용자에게 실행 중인 리소스 사용량 및 성능 특징을 이해하게끔 도와줍니다 {{< glossary_tooltip text="컨테이너" term_id="container" >}}
+
+<!--more-->
+
+It is a running daemon that collects, aggregates, processes, and exports information about running containers. Specifically, for each container it keeps resource isolation parameters, historical resource usage, histograms of complete historical resource usage and network statistics. This data is exported by container and machine-wide.
```
Comment (bart2001) [inline content/ko/docs/reference/glossary/cadvisor.md:18]: 영어 원문 삭제가 필요합니다.

**Comment by seokho-son** [comment]
reopened #36744
/close

**Pull request closed without merging**